shared/AppInsightsCommon/src/Interfaces/IConfig.ts

```diff
@@ -389,6 +389,12 @@ export interface IConfig {
      * This URL takes precedence over the 'config.endpointUrl' and any endpoint in the connection string.
      */
     userOverrideEndpointUrl?: string;
+
+    /**
+     * [Optional] If set to true, when exception is sent out, the SDK will also send out all scripts basic info that are loaded on the page.
+     * Notice: This would increase the size of the exception telemetry.
+     */
+    getExceptionScriptsInfo?: boolean;
```

Comment:
Rather than "get" (which to me sounds like a function) we want to call out that this is an option, which is where we have in the past called these "enableXXXX" or "disableXXXX" in this case maybe something like "includeXXX" and as we are also adding the debug logs and maybe some other stuff for the internal project, perhaps we call these out into an exception config (like the Cookie manager config)

So something like

```ts
expCfg: {
    inclScripts: true/false,
    getDbgLogs: () => string[]
}
```

We can call the interface name for this something meaningful like `IExceptionConfig` as I suspect that we are going to need additional optional "values" that partners will need to include.